src/main/scala/com.snowplowanalytics.sauna/observers/AmazonKinesisObserver.scala

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) 2016 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+package com.snowplowanalytics.sauna
+package observers
+
+// scala
+import scala.util.control.NonFatal
+import scala.concurrent.duration._
+import scala.concurrent.{ Future, ExecutionContext }
+import ExecutionContext.Implicits.global
+
+// java
+import java.net.URLDecoder
+
+// akka
+import akka.actor._
+
+// play
+import play.api.libs.json.{ Json, JsValue }
+
+// amazonaws
+import com.amazonaws.auth.{ BasicAWSCredentials, AWSCredentialsProvider, DefaultAWSCredentialsProviderChain }
+import com.amazonaws.AbortedException
+import com.amazonaws.services.kinesis.clientlibrary.lib.worker.KinesisClientLibConfiguration
+import com.amazonaws.services.kinesis.model.Record
+
+//kinesis
+import com.gilt.gfc.aws.kinesis.client.{ KCLConfiguration, KinesisRecordReader, KCLWorkerRunner }
+
+// sauna
+import responders.Responder._
+import Observer.{ KinesisRecordReceived }
```

Comment:
Some of these imports aren't used in the code - here's a list of _used_ imports as optimized by IntelliJ IDEA:

```scala
// scala
import scala.concurrent.ExecutionContext.Implicits.global
import scala.concurrent.Future
import scala.concurrent.duration._

// akka
import akka.actor._

// amazonaws
import com.amazonaws.auth.{AWSCredentialsProvider, BasicAWSCredentials, DefaultAWSCredentialsProviderChain}
import com.amazonaws.services.kinesis.clientlibrary.lib.worker.KinesisClientLibConfiguration
import com.amazonaws.services.kinesis.model.Record

// kinesis
import com.gilt.gfc.aws.kinesis.client.{KCLConfiguration, KCLWorkerRunner, KinesisRecordReader}

// sauna
import com.snowplowanalytics.sauna.observers.Observer.KinesisRecordReceived
```